scripts/01_prep_inputs/0100-extract-inputs.R

```diff
@@ -828,15 +821,15 @@ hab_fish_input <- left_join(hab_fish_input_prep2,
   # Order like template
   relocate(temperature_c, conductivity_m_s_cm, turbidity, .after = site) |>
   relocate(local_name, .after = reference_number) |>
+  relocate(fish_activity, .after = reference_number) |>
```

Comment:
seems a bit odd from uninformed glance... Next time let's put a note why this new addition seemed necessary

Reply:
okay sounds good. Added it to match column order in spreadsheet for easier copy/paste